security/compliance_operator/compliance-operator-crd.adoc

```diff
@@ -0,0 +1,509 @@
+<2> Specify the XCCDF identifier of the profile that you want to run.
+<3> Specify the container image that encapsulates the profile files.
+<4> It is optional. Specify the scan to run a single rule. This rule has to be identified with the XCCDF ID, and has to belong to the specified profile.
++
+[NOTE]
+====
+If you skip the `rule` parameter, then scan runs for all the available rules of the specified profile.
+====
+<5> If you are on the {product-title} and wants to generate a remediation, then nodeSelector label has to match the `MachineConfigPool` label.
++
+[NOTE]
+====
+If you do not specify `nodeSelector` parameter or match the `MachineConfig` label, scan will still run, but it will not create remediation.
+====
+<6> Indicates the current phase of the scan.
+<7> Indicates the verdict of the scan.
+
+[IMPORTANT]
+====
+If you delete a `ComplianceSuite` object, then all the associated scans get deleted.
+====
+
+When the scan is completed, it generates the result as Custom Resources of the `ComplianceCheckResult` object. However, the raw results are available in ARF format. These results are stored in a Persistent Volume (PV), which has a Persistent Volume Claim (PVC) associated with the name of the scan.
+You can programmatically fetch the `ComplianceScans` events. To generate events for the suite, run the following command:
+
+[source,terminal]
+----
+oc get events --field-selector involvedObject.kind=ComplianceScan,involvedObject.name=<name of the suite>
+----
+
+[id="view-compliance-results_{context}"]
+== Viewing the compliance results
+When the compliance suite reaches the `DONE` phase, you can view the scan results and possible remediations.
+
+[id="compliance-check-result_{context}"]
+=== ComplianceCheckResult object
+When you run a scan with a specific profile, several rules in the profiles are verified. For each of these rules, a `ComplianceCheckResult` object gets created, which provides the state of the cluster for a specific rule.
+
+.Example `ComplianceCheckResult` object
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: ComplianceCheckResult
+metadata:
+  labels:
+    compliance.openshift.io/check-severity: medium
+    compliance.openshift.io/check-status: FAIL
+    compliance.openshift.io/suite: example-compliancesuite
+    compliance.openshift.io/scan-name: workers-scan
+  name: workers-scan-no-direct-root-logins
+  namespace: openshift-compliance
+  ownerReferences:
+  - apiVersion: compliance.openshift.io/v1alpha1
+    blockOwnerDeletion: true
+    controller: true
+    kind: ComplianceScan
+    name: workers-scan
+description: <description of scan check>
+instructions: <manual instructions for the scan>
+id: xccdf_org.ssgproject.content_rule_no_direct_root_logins
+severity: medium <1>
+status: FAIL <2>
+----
+
+<1> Describes the severity of the scan check.
+<2> Describes the result of the check. The possible values are:
+* PASS: indicates the check was successful.
```

Comment:
For brevity, you could drop "indicates" in this list.

Reply:
Done